composer.json

```diff
@@ -12,9 +12,9 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "illuminate/support": "^5.1",
-        "illuminate/database": "^5.1",
-        "illuminate/events": "^5.1"
+        "illuminate/support": "^5.1|^6.0",
```

Comment:
perhaps this should be changed to `^5.8|^6.0` if I am not mistaken.

Reply:
Actually, it should be 6.0 only, as 5.x does not support DeferrableProvider for service providers.

Reply:
yes but the package shouldn't be installed for anyone having `<5.8` since it will break for them.

Reply:
I'll change it then.